pkg/udev/build.yml

```diff
@@ -0,0 +1,15 @@
+image: eve-udev
+org: lfedge
+config:
+  binds:
+    - /:/hostfs
+    - /dev:/dev
+    - /run:/run
+    - /lib/modules:/lib/modules
+  capabilities:
+    - all
+  net: host
```

Comment:
Why do you need network access to build this? We want this to be disabled to avoid accidentally adding wget/curl/git pull to the Docker files since the SBoM generation would not notice such downloads (ADD commands are detected and make it into the SBoM but ADD does not need this network access.)

Reply:
nice catch, it wasn't intended to be there, I've removed...